




	
    <div class="row">
		<div class="col-md-10 col-md-offset-1">
			<div class="panel panel-default">
				<div class="panel-body">
					<div class="h3"><i style="color:#F87217">About the PeptoVarDB project</i></div>
					<br>
					<p style="text-align:justify">
PeptoVarDB is an on-line database of annotated peptide based on human genome annotation and public data of human genome common variants created with PeptoVar program in the pupulation mode.
PeptoVar (Peptides of Variations) is a program for annotation of genomic variants in protein coding genes and generation of annotated peptides (<a href="https://open-projects.github.io/PeptoVar/">open-projects.github.io/PeptoVar/</a>).
The main idea of PeptoVar is implementation of the potential compound effects of combining alternate alleles across multiple variant loci in sequence translation.
PeptoVar is helpful for in-silico prediction of new minor histocompatibility antigen (MiHa), simulation of stem sells transplantation 
or as peptide database generator for shotgun proteomic studies of immunopeptidome.
<br>
The current release of PeptoVarDB based on RefSeq Annotation release 108 (<a href="https://www.ncbi.nlm.nih.gov/refseq/">www.ncbi.nlm.nih.gov/refseq/</a>) and dbSNP Build 151 (<a href="https://www.ncbi.nlm.nih.gov/projects/SNP/">www.ncbi.nlm.nih.gov/projects/SNP/</a>).
Only genome variants with Allele Frequency (AF) > 0.01 was used as the input data for the PeptoVar program. 
The program was started with the next parameters: <br>
<i>PeptoVar.py -tagaf CAF -minaf 0.01 -peptlen 8 9 10 11 12 -var nonsyn -indir ./input </i>
</p>

				</div>
			</div>
        	</div>
    </div>
